packages/fxa-content-server/app/scripts/lib/constants.js

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const popularEmailDomains = require('./popular-email-domains.json');
+const popularEmailDomains = require('../../../../fxa-shared/email/popularDomains.json');
```

Comment:
`popular-email-domains.json` was a symlink. We could have kept it around but it seemed unnecessary now we can just pull in the actual file directly.